libbeat/generator/fields/module_fields_collector.go

```diff
@@ -71,7 +71,7 @@ func CollectModuleFiles(modulesDir string) ([]*YmlFile, error) {
 func CollectFiles(module string, modulesPath string) ([]*YmlFile, error) {
 
 	var files []*YmlFile
-	fieldsYmlPath := filepath.Join(modulesPath, module, "_meta", "fields.yml")
+	fieldsYmlPath := filepath.Join(modulesPath, module, "_meta/fields.yml")
```

Comment:
What is the reason you introduced the `/` here? Could break on Windows. Any issues with the previous option?

Reply:
Andrew pointed out that it is working everywhere in https://github.com/elastic/beats/pull/7713#discussion_r205152643